pkg/kafka/auth/options.go

```diff
@@ -49,6 +49,21 @@ const (
 
 	defaultPlainTextUserName = ""
 	defaultPlainTextPassword = ""
+
+	// Scram configuration options
+	scramPrefix                   = ".scram"
+	suffixScramUserName           = ".username"
+	suffixScramAlgorithm          = ".algorithm"
+	suffixScramPassword           = ".password"
+	suffixScramTLS                = ".usetls"
```

Comment:
there is already `kafka.tls.enable` flag, how is this one going to be different?

Reply:
@pavolloffay this might be a use case for keeping `kafka.tls.enable`

Reply:
Changed to just use whatever tls configuration the saramaConfig is using.

Reply:
why do we need to keep this flag? We already have the same signal from `--kafka.producer.tls.enabled`

Reply:
Removed!